cfile/hogfile.h

```diff
@@ -62,9 +62,11 @@
 
 #include <cstdint>
 #include <cstdio>
+#include <string>
```

Comment:
This include unneeded.

Reply:
Good catch. I just force pushed a new version that doesn’t have that line.